tests/Api/ProductTest.php

```diff
@@ -26,7 +26,7 @@ public function test_can_call_magento_api_products_show()
     {
         Http::fake();
 
-        $api = MagentoFacade::api('products')->show('foo');
+        $api = MagentoFacade::api('products')->getBySku('foo');
```

Comment:
¡Ay caramba!
I didn't realize that `Products::show()` was renamed to `Products::getBySku()`.

On the other hand, it is kind of funny that the test for `Products::all()` doesn't use any arguments and thus doesn't run afoul of the breaking changes to the order of arguments in the ID fork. 😄

Reply:
@developh 
Please discard the changes to this test because the order of arguments for `Products::show()` was restored in `main` to remain compatible with the original package and therefore this test will fail if merged into `main`.